### PR TITLE
[inbound roaming] Feature docs

### LINKED ISSUE #5729: [Inbound Roaming][doc] Add inbound roaming documentation

### PR DESCRIPTION
**Pull request by uri200**
Signed-off-by: Oriol Batalla <obatalla@fb.com>

## Summary

Doc for inbound roaming setup

## Test Plan

image

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/howtos/inbound_roaming.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+title: Inbound Roaming
+hide_title: true
+---
+
+*Last Updated: 4/08/2021*
+
+# Inbound Roaming
+
+Inbound Roaming allows subscribers belonging to other operators (roaming
+subscribers) to use Magma served network.
```
Comment (themarwhal) [inline docs/readmes/howtos/inbound_roaming.md]: to use a Magma served network. ?

File: docs/readmes/howtos/inbound_roaming.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+Inbound Roaming requires Magma operator to reach agreements with
+other operators to have direct connectivity to their HSS through `S6a`
+(diameter) and PGW through `S8` (gtp) interfaces. VPN is suggested to reach
+those roaming services but is out of scope of Magma.
+
+## Architecture
+
+Currently, we support two Architectures:
+- **Non Federated + Roaming**: where local subscribers are stored in
+  SubscriberDb and roaming subscribers use a remote Federated Gateway to reach HSS/PGW
```
Comment (themarwhal) [inline docs/readmes/howtos/inbound_roaming.md]: I think we usually spell subscriberdb -> `SubscriberDB`

File: docs/readmes/howtos/inbound_roaming.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+Inbound Roaming requires Magma operator to reach agreements with
+other operators to have direct connectivity to their HSS through `S6a`
+(diameter) and PGW through `S8` (gtp) interfaces. VPN is suggested to reach
+those roaming services but is out of scope of Magma.
+
+## Architecture
+
+Currently, we support two Architectures:
+- **Non Federated + Roaming**: where local subscribers are stored in
```
Comment (themarwhal) [inline docs/readmes/howtos/inbound_roaming.md]: `Non Federated` -> `Non-federated`? maybe? not sure about this one.

File: docs/readmes/howtos/inbound_roaming.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+requires a `Federated` setup to work. So the to configure Inbound Roaming
+you need:
+- Functioning [Or8cr](https://docs.magmacore.org/docs/orc8r/architecture_overview),
+- Install [Federatetion Gateway](https://docs.magmacore.org/docs/feg/deploy_intro) and,
+- Install [Access Gateway](https://docs.magmacoreorg/docs/lte/setup_deb).
+- Configure it as a [Federated Deployment](https://docs.magmacore.org/docs/feg/federated_FWA_setup_guide)
+- Make sure your setup is able to serve calls with your local subscribers
+  (in case of using subscriber DB you will need to set `"hss_relay_enabled":
+  false` on Federated LTE Network temporally to test this. Set it back to
+  `True` once tested!!)
```
Comment (themarwhal) [inline docs/readmes/howtos/inbound_roaming.md]: Lower case `true` for consistency

File: docs/readmes/howtos/inbound_roaming.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+
+At the end of this you will have:
+- A Federated LTE Network with an LTE Gateway in it (AGW)
+- A Federation Network with a Federation Gateway in it (FEG)
```
Comment (themarwhal) [inline docs/readmes/howtos/inbound_roaming.md]:
```
- A Federated LTE Network with an LTE Gateway (AGW)
- A Federation Network with a Federation Gateway (FEG)
```

File: docs/readmes/howtos/inbound_roaming.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+
+We will refer to them as `Local` Networks and Gateways to differentiate them
```
Comment (themarwhal) [inline docs/readmes/howtos/inbound_roaming.md]: `Local` and `roaming` casing should be consistent

Comment (uri200) [reply docs/readmes/howtos/inbound_roaming.md]: all lowered cased, except for `Inbound Roaming`

**Review by themarwhal — APPROVED** [review]
I think it looks good? :D

**Pull request merged**